specification/contosowidgetmanager/Contoso.WidgetManager/tspconfig.yaml

```diff
@@ -37,3 +37,11 @@ options:
     package-dir: "azure-contoso-widgetmanager"
     namespace: com.azure.contoso.widgetmanager
     flavor: azure
+  "@azure-tools/typespec-go":
+    module: "github.com/Azure/azure-sdk-for-go/{service-dir}/{package-dir}"
+    service-dir: "sdk/contosowidget"
+    package-dir: "azmanager"
+    module-version: "0.0.1"
```

Comment:
if this a must for data plane?

Reply:
When `module` is set, `module-version` must also be set (this is true regardless of input type).

Reply:
we did not config it for control plane, see [here](https://github.com/Azure/azure-rest-api-specs/blob/a26af9acb30387724eff64229ba7f3259600681a/specification/azurefleet/AzureFleet.Management/tspconfig.yaml#L39). do we have special logic for control plane in go emitter?

Reply:
i see. thanks for confirming.

Reply:
what is module-version?

Reply:
it is used in go codegen to determine package version. it is specific for go package.